css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
+	background-size: contain;
+	margin-left: 50px;
+	margin-top: 20px;
+	float: left;
+	margin-right: 150px;
+}
+
+header{
+	width: 100%;
+	height: 80px;
+	box-shadow: 0px 3px 10px grey;
```

Comment:
Would `0%` work better than `0px` here? What about `0em`? Would there be any difference?